init-interfaces.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -ex
+
+is_con_exists() {
+	local con_name=$1
+	if [[ $(nmcli -t -f NAME con show | grep -w "$con_name") ]]; then
+		echo "true"
```

Comment:
Why not just exit 0 vs exit 1? That is the idiomatic way to deal with booleans in bash AFAIK

Reply:
I think I should have used `return n`. tell me what you think

Reply:
You are right, exit can't be used in function.

What about just dropping the if and only calling the content? It already returns the exit code.

Reply:
but I still need to return 0/1, no?
just calling the content in the function won't return the value of it IIUC (nor doing `return $(func)`)
I changed it so it works now